src/transform/join.js

```diff
@@ -57,7 +57,7 @@ Transform.define("join", function(pos, depth = 1) {
   for (let i = 0; i < depth; i++) {
     let $pos = this.doc.resolve(pos)
     if ($pos.parentOffset == 0 || $pos.parentOffset == $pos.parent.content.size)
-      return this.fail(new Error("Nothing to join at " + pos))
+      return this.fail("Nothing to join at " + pos)
```

Comment:
Oh, right. It was all error objects at one point, but I guess somewhere in that thicket of refactors I changed that. Thanks.

(Though this whole API still feels like unnecessary complexity and I'm trying to find a way to get rid of it.)